jupyterlab_git/git.py

```diff
@@ -609,6 +636,7 @@ async def detailed_log(self, selected_hash, path):
 
         return {
             "code": code,
+            "commit_body": strip_and_split(my_commit_output.split("\n", 1)[-1]),
```

Comment:
Why are you keeping only the last item?

If this is to drop the commit summary, is it not enough to use `%b` instead of `%B` in the format.